### PR TITLE
gbXML 7.03

### PR DESCRIPTION
**Pull request by MatthewSteen**
Pull request overview
---------------------

> *WIP to deliver a custom build to Autodesk prior to inclusion in a major release.*

### Pull Request Author

 - [ ] Model API Changes / Additions
 - [ ] Any new or modified fields have been implemented in the EnergyPlus ForwardTranslator (and ReverseTranslator as appropriate)
 - [ ] Model API methods are tested (in `src/model/test`)
 - [ ] EnergyPlus ForwardTranslator Tests (in `src/energyplus/Test`)
 - [ ] If a new object or method, added a test in NREL/OpenStudio-resources: Add Link
 - [ ] If needed, added VersionTranslation rules for the objects (`src/osversion/VersionTranslator.cpp`)
 - [ ] Verified that C# bindings built fine on Windows, partial classes used as needed, etc.
 - [ ] All new and existing tests passes
 - [ ] If methods have been deprecated, update rest of code to use the new methods

**Labels:**

 - [ ] If change to an IDD file, add the label `IDDChange`
 - [ ] If breaking existing API, add the label `APIChange`
 - [ ] If deemed ready, add label `Pull Request - Ready for CI` so that CI builds your PR

### Review Checklist

This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] Code Style, strip trailing whitespace, etc.
 - [ ] All related changes have been implemented: model changes, model tests, FT changes, FT tests, VersionTranslation, OS App
 - [ ] Labeling is ok
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified

### CODE REVIEW

File: src/utilities/xml/resources/GreenBuildingXML_Ver7.03.xsd [hunk 1]
```diff
@@ -0,0 +1 @@
...
		</xsd:annotation>
		<xsd:complexType>
			<xsd:simpleContent>
				<xsd:extension base="xsd:decimal">
					<xsd:attribute name="unit" type="unitlessUnitEnum" use="required"/>
				</xsd:extension>
			</xsd:simpleContent>
		</xsd:complexType>
	</xsd:element>
	<xsd:element name="P
```
Comment (jmarrec) [inline src/utilities/xml/resources/GreenBuildingXML_Ver7.03.xsd:1]: Seems like there are line ending issues here.

File: src/utilities/xml/XMLValidator.cpp [hunk 2]
```diff
@@ -477,8 +477,8 @@ XMLValidator XMLValidator::gbxmlValidator() {
     LOG_AND_THROW("Failed to create a temporary directory for extracting the embedded path");
   }
   const bool quiet = true;
-  ::openstudio::embedded_files::extractFile(":/xml/resources/GreenBuildingXML_Ver6.01.xsd", openstudio::toString(tmpDir), quiet);
-  return XMLValidator(tmpDir / "GreenBuildingXML_Ver6.01.xsd");
+  ::openstudio::embedded_files::extractFile(":/xml/resources/GreenBuildingXML_Ver7.03.xsd", openstudio::toString(tmpDir), quiet);
```
Comment (jmarrec) [inline src/utilities/xml/XMLValidator.cpp:480]: Maybe we should use an enum (or a string with validation... ) and modifying the gbxmlValidator signature to take en optional gbxmlVersion parameter?

```c++
enum class GbxmlVersion { Ver601, Ver703 };
XMLValidator gbxmlValidator(GbxmlVersion gbxmlVersion = GbxmlVersion::Ver601);
```

The source code in `src/gbxml` needs to be modified to match as well.

Comment (jmarrec) [reply src/utilities/xml/XMLValidator.cpp:480]: https://github.com/NREL/OpenStudio/blob/950172bc1247f6059a48d5ec1c314fab283a18de/src/gbxml/ForwardTranslator.cpp#L185-L191

**Comment by MatthewSteen** [comment]
Closing and trying again. Base branch was supposed to be `3.6.0` and I mistakenly merged develop into this branch.

**Pull request closed without merging**